modules/xmpp/TraceablePeerConnection.js

```diff
@@ -322,11 +321,8 @@ TraceablePeerConnection.prototype.addStream = function (stream, ssrcInfo) {
     }
 };
 
-TraceablePeerConnection.prototype.removeStream = function (stream, stopStreams) {
+TraceablePeerConnection.prototype.removeStream = function (stream) {
     this.trace('removeStream', stream.id);
-    if (stopStreams) {
-        RTC.stopMediaStream(stream);
```

Comment:
I searched for usages and the method is never called with true as the second argument. I don't know if it was used before. Maybe I better double check on that.